diff --git a/index.js b/index.js
index c3f5471eebdd84fd34f95c03da8eee663c411130..e560911c0d844532c70f48b65a7bf39d8b4db05a 100644
--- a/index.js
+++ b/index.js
@@ -4,7 +4,7 @@ const { VirtualFileSystem } = require('./lib/file_system.js');
 const { VirtualProvider } = require('./lib/provider.js');
 const { MemoryProvider } = require('./lib/providers/memory.js');
 const { RealFSProvider } = require('./lib/providers/real.js');
-const { SqliteProvider } = require('./lib/providers/sqlite.js');
+// const { SqliteProvider } = require('./lib/providers/sqlite.js');
 
 /**
  * Creates a new VirtualFileSystem instance.
@@ -31,5 +31,5 @@ module.exports = {
   VirtualProvider,
   MemoryProvider,
   RealFSProvider,
-  SqliteProvider,
+  // SqliteProvider,
 };
